dev-requirements.txt

```diff
@@ -17,7 +17,9 @@ Sphinx==1.6.5
 sphinx_rtd_theme
 flasgger==0.9.1
 -e git+https://git@github.com/uc-cdis/cdisutils-test.git@0.0.1#egg=cdisutilstest
--e git+https://git@github.com/uc-cdis/indexd.git@1.0.8#egg=indexd
+-e git+https://git@github.com/uc-cdis/indexd.git@1.1.2#egg=indexd
```

Comment:
you made some bug fixes in indexd after 1.1.2, if they are relevant to this PR let's create a new tag?

Reply:
yes! will update tags before mergine